scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -746,9 +746,9 @@ private[scalatest] class RunnerJFrame(
 
       val itemAction: AbstractAction =
         new AbstractAction(menuItemText) {
-          def actionPerformed(ae: ActionEvent): Unit = {
+          def actionPerformed(ae: ActionEvent | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.